src/index.css

```diff
@@ -6,7 +6,7 @@
 
   color-scheme: light dark;
   color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
```

Comment:
You can remove all styles and only import `antd` styles `@import "antd/dist/reset.css";`